apps/next-evans/lib/friend.ts

```diff
@@ -14,8 +14,16 @@ export function getFriendUrl(friendSlug: string, gender: Gender): string {
   return `/${firstPart}/${friendSlug}`;
 }
 
-export function getDocumentUrl(friendSlug: string, documentSlug: string): string {
-  return `/${friendSlug}/${documentSlug}`;
+export function getDocumentUrl(doc: { authorSlug: string; slug: string }): string;
+export function getDocumentUrl(str1: string, str2: string): string;
```

Comment:
this function signature will show up in our editors intellisense, so we want to preserve good names for variables, so these should still be `friendSlug` and `documentSlug`. the implementation can remain the same, it's effectively hidden to us at the call site.